src/main/kotlin/week1/racingcar/domain/Car.kt

```diff
@@ -1,17 +1,16 @@
-package week1.racingcar
-
-import kotlin.random.Random
+package week1.racingcar.domain
 
 class Car(
     val name: String,
+    private var mileage: Int = DEFAULT_MILEAGE,
 ) {
-    var mileage = DEFAULT_MILEAGE
-        private set
 
     init {
         validateCarInfo()
     }
 
+    fun currentMileage() = mileage
```

Comment:
메서드를 만드시는 것도 괜찮지만 아래와 같이 읽기전용 프로퍼티를 만드시는 것도 괜찮은 방식 같아요. 😃

```suggestion
    val currentMileage
        get() = mileage
    
    init {
        validateCarInfo()
    }
```

이렇게 작성했을 때, java로 변경해보면 어떻게되는지 decompile 해보시면 좋을 것 같아요. 😉
<img width="1510" alt="image" src="https://github.com/next-step/kotlin-racingcar/assets/47850258/36f8397e-263b-4492-9618-2b34d6d32f1f">

Reply:
역할이 딱히 없어서 좀 망설였던 부분인데, 읽기 전용 프로퍼티로 만드는게 보기가 더 좋은 것 같네요!
동작은 완벽히 같군요